packages/react/src/internal/utils/getInteractiveNodes.ts

```diff
@@ -0,0 +1,23 @@
+const interactiveElements = [
```

Comment:
One package we could use for this: https://www.npmjs.com/package/focusable-selectors (either as a dependency or inline with accreditation) that has a pretty robust list 👀

Reply:
Definitely! Added some more selectors that could be handy.